easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -318,8 +318,29 @@ def configure_step(self, srcdir=None, builddir=None):
 
         (out, _) = run_cmd(command, log_all=True, simple=False)
 
-        return out
-
+        # sanitycheck for the configuration step
+        self.log.info("Checking python paths")
+        with open('CMakeCache.txt', 'r') as file:
+            lines = file.readlines()
+
+        # Search for Python paths in each line
+        for line in lines:
+            if line.startswith('Python3_EXECUTABLE:FILEPATH='):
+                pythonExecPath = line.split('=')[1].strip()
+            elif line.startswith('Python3_INCLUDE_DIR:FILEPATH='):
+                pythonIncludePath = line.split('=')[1].strip()
+            elif line.startswith('Python3_LIBRARY:FILEPATH='):
+                pythonLibraryPath = line.split('=')[1].strip()
+        self.log.info("Python executable path: %s", pythonExecPath)
+        self.log.info("Python include path: %s", pythonIncludePath)
+        self.log.info("Python library path: %s", pythonLibraryPath)
+        # Check if paths include EBROOTPYTHON
+        ebrootpython_path = os.getenv('EBROOTPYTHON')
+        if all(path and ebrootpython_path in path for path in [pythonExecPath, pythonIncludePath, pythonLibraryPath]):
```

Comment:
See above: `pythonExecPath` & co might not have been set at all. And maybe `path.startswith(ebrootpython_path)`? We possibly need to take symlinks into account too: either of the 2 might be a resolved symlink or not.